tests/angular_devkit/build_ng_packagr/ng-packaged/projects/lib/tsconfig.lib.json

```diff
@@ -21,7 +21,8 @@
     "fullTemplateTypeCheck": true,
     "strictInjectionParameters": true,
     "flatModuleId": "AUTOGENERATED",
-    "flatModuleOutFile": "AUTOGENERATED"
+    "flatModuleOutFile": "AUTOGENERATED",
+    "enableResourceInlining": true
```

Comment:
We need to properly communicate that if consumers are using `ng-packagr v3` when they update to `v4` they need to add this flag in their existing tsconfig.

Reply:
WDYT of checking that this field is missing and adding a warning/error at runtime?

Reply:
I was actually thinking of switching this option to true by default in ng-packagr

Reply:
That works too, and seems like you have already 👍 https://github.com/dherges/ng-packagr/pull/1021

Reply:
yeah :)